### PR TITLE
Created a guide for Restoring Accounts

### LINKED ISSUE #592: Create guide for address recovery (on polkadot.js, polkadot.js plugin, Parity Signer)
We have walkthrough guides for CREATING an address but not RECOVERING one.  We should add these in with screenshots.

### PR DESCRIPTION
**Pull request by krichard410**
Closes #592

### CODE REVIEW

File: docs/learn-account-restore.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+that your account has been restored. That's it!
+
+![restore-complete-polkadot](assets/accounts/polkadot_restore_complete.png)
+
+## Parity Signer
+
+To restore an account on Parity Signer, open the app and click on the user profile icon in the top
+right hand corner.
+
+![click-profile-parity](assets/accounts/parity_click_profile.png)
```
Comment (krichard410) [inline docs/learn-account-restore.md]: Does anyone know if there a way to resize markdown photos? Some of these photos are huge!

Comment (ansonla3) [reply docs/learn-account-restore.md]: maybe try to use img tag with width option?

Comment (lsaether) [reply docs/learn-account-restore.md]: You can also size the photo in an image editor.

File: docs/learn-account-restore.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+In the case that your Polkadot account needs to be restored, there are simple ways to take care of
+this in Polkadot.js Browser Plugin, Polkadot.js, and Parity Signer. Note that in order to recover an
+account, it is necessary that an account is made according to the instructions
+[here](learn-account-generation).
+
+## Polkadot.js Browser Plugin
+
+For using the Polkadot.js browser plugin, restoring an account is simple. Let's start off by heading
+over to the extension and selecting on the desired account to restore. Click on the three dots
```
Comment (lsaether) [inline docs/learn-account-restore.md]:
```suggestion
over to the extension and selecting the desired account to restore. Click on the three dots
```

**Review by lsaether — COMMENTED** [review]
Can you blur the seed phrases in the images? It's best to not mislead the reader that it's generally okay to share these.

**Comment by krichard410** [comment]
> Can you blur the seed phrases in the images? It's best to not mislead the reader that it's generally okay to share these.

Sure! It might be a good idea too if the seed phrases images were blurred out on https://wiki.polkadot.network/docs/en/learn-account-generation#docsNav 
to keep the site consistent

**Review by lsaether — APPROVED** [review]
Yes we should blur those images as well in future PRs.

**Comment by Swader** [comment]
Looks great, this is ready for review?

**Comment by krichard410** [comment]
> Looks great, this is ready for review?

Yup! All ready

File: docs/learn-account-restore.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+In the case that your Polkadot account needs to be restored, there are simple ways to take care of
+this in Polkadot.js Browser Plugin, Polkadot.js, and Parity Signer. Note that in order to recover an
+account, it is necessary that an account is made according to the instructions
+[here](learn-account-generation).
+
+## Polkadot.js Browser Plugin
+
+For using the Polkadot.js browser plugin, restoring an account is simple. Let's start off by heading
+over to the extension and selecting the desired account to restore. Click on the three dots beside
```
Comment (Swader) [inline docs/learn-account-restore.md:16]: It seems a little strange to me to start this document talking about account restoration, and then go into the process of creating a backup. Maybe replace `to restore` here with `to back up and later restore`.

File: docs/learn-account-restore.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+the address to open up a settings tab.
+
+![list-accounts-plugin](assets/accounts/polkadot.js_list_accounts.png)
+
+From here, we will be selecting the Export Account button, which will then ask you for the password
+for that specific account. Once the password is entered, the browser will automatically download a
```
Comment (Swader) [inline docs/learn-account-restore.md]: Consider using active tense across the post. "Once you enter the password".

File: docs/learn-account-restore.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+.json file that will hold all the account restoration details that will be used when you restore
```
Comment (Swader) [inline docs/learn-account-restore.md]:
```suggestion
`.json` file that will hold all the account restoration details that will be used when you restore
```

File: docs/learn-account-restore.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+![restore-account-plugin](assets/accounts/polkadot.js_restore_account.png)
+
+Once these are filled out, and the "Restore" button has been pressed, you'll be taken back to the
+main page of the plugin. There listed will be your restored account!
+
+## Polkadot.js
+
+If you're using the main Polkadot.js UI, restoring an account will feel similar to restoring an
+account on Polkadot.js browser plugin. The first step is to head over to the chosen account and
+click on the three dots beside it. This will open up a tab with lots of options. For recovering an
```
Comment (Swader) [inline docs/learn-account-restore.md]: Same as above, I'd say "for backing up the account we'll restore later..."

File: docs/learn-account-restore.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+account, press on "Create a backup file for this account".
+
+![click-account-restore-polkadot](assets/accounts/polkadot_click_restore.png)
+
+From there, a pop up screen will appear, prompting for the password of that account. By entering
+your password and clicking on "Download", your browser will download the .json file associated with
```
Comment (Swader) [inline docs/learn-account-restore.md]:
```suggestion
your password and clicking on "Download", your browser will download the `.json` file associated with
```

File: docs/learn-account-restore.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+the Polkadot account. This .json file holds all relevant data about the account to be used in
```
Comment (Swader) [inline docs/learn-account-restore.md]:
```suggestion
the Polkadot account. This `.json` file holds all relevant data about the account to be used in
```

File: docs/learn-account-restore.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+account restoration.
+
+![enter-account-password-polkadot](assets/accounts/polkadot_enter_password.png)
+
+Now that's completed, it's time to restore the account. On the Polkadot accounts page, click on the
+"Restore JSON" button, which will let you upload your .json file that you downloaded and enter your
```
Comment (Swader) [inline docs/learn-account-restore.md]:
```suggestion
"Restore JSON" button, which will let you upload your `.json` file that you downloaded and enter your
```

File: docs/learn-account-restore.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+password for that account.
+
+![restore-account-polkadot](assets/accounts/polkadot_restore_account.png)
+
+Once the "Restore" button has been pressed, you should see a green notification letting you know
```
Comment (Swader) [inline docs/learn-account-restore.md]: Once you've pressed...

File: docs/learn-account-restore.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+that your account has been restored. That's it!
+
+![restore-complete-polkadot](assets/accounts/polkadot_restore_complete.png)
+
+## Parity Signer
+
+To restore an account on Parity Signer, open the app and click on the user profile icon in the top
```
Comment (Swader) [inline docs/learn-account-restore.md]:
```suggestion
To restore an account in Parity Signer, open the app and click on the user profile icon in the top
```

Comment (Swader) [reply docs/learn-account-restore.md]: Again I'd say "to back up an account we'll restore below..."

File: docs/learn-account-restore.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+Now, it's time to restore the account. Open up the Polkadot.js on your browser, and navigate to "Add
+Account". Though this may seem counter-intuitive, we aren't actually adding in an account here,
+rather we are restoring the account using its mnemonic recovery phrase. It is very important here to
+put in the mnemonic recovery phrase from your Parity Signer account.
+
+Note that you can change the name and password if desired.
+
+![restoring-account-parity](assets/accounts/parity_restoring_account.jpg)
+
+Once you hit save, it is important to "Create and backup account", which will download a .json file
```
Comment (Swader) [inline docs/learn-account-restore.md]:
```suggestion
Once you hit save, it is important to "Create and backup account", which will download a `.json` file
```

**Review by Swader — CHANGES_REQUESTED** [review]
Minor nitpicks. I'm also wondering if we should cover social recovery here too, or if that should be another doc?

**Pull request merged**